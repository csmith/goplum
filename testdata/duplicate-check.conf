check debug.random "same" {
    interval = 3s
    good_threshold = 5
    failing_threshold = 6
}

check http.get "same" {}

alert debug.sysout "text" {}
