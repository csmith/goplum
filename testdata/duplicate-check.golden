"check defined multiple times: same"